src/main/java/com/taskadapter/redmineapi/IssueManager.java

```diff
@@ -9,10 +9,7 @@
 import com.taskadapter.redmineapi.bean.SavedQuery;
 import com.taskadapter.redmineapi.bean.Tracker;
 import com.taskadapter.redmineapi.bean.Watcher;
-import com.taskadapter.redmineapi.internal.DirectObjectsSearcher;
-import com.taskadapter.redmineapi.internal.Joiner;
-import com.taskadapter.redmineapi.internal.ResultsWrapper;
-import com.taskadapter.redmineapi.internal.Transport;
+import com.taskadapter.redmineapi.internal.*;
 import org.apache.http.NameValuePair;
```

Comment:
let's avoid using wildcard imports here as well as in other places, those bring random classes at times.